guide/README.md

```diff
@@ -0,0 +1,285 @@
+
+Input files
+
+Here is an example of the ESPResSo++ input files to run the CG MD simulations of the Iterative Boltzmann inversion procedure.
+
+  * `conf.gro` - stores the coordinates of the molecule(s). Can be viewed with vmd. A pdb or xyz file would be okay, too.
+  * `spce.py` is the simulation script which will be called by csg_inverse. It stores the whole simulation procedure.
+  * `topol.xml` -topology of the molecule, defined in the initial condition. This is needed as most gro/pdb/xyz files have not molecule definition in them.
+
+
+# Mapping an atomistic onto a coarse-grained trajectory
+
+## SPC/E Water
+
+Folder: `csg-tutorials/spce/atomistic/`
+
+Have a look at the center of mass mapping file `water.xml` in this folder. Use `csg_map` to create a coarse-grained configuration (adjust the corresponding command the `run.sh` script to map `conf.gro` instead of `confout.gro`). Visualize both configurations with vmd. (execute `vmd conf.gro &`)
+
+Create a mapping file where the center of a coarse-grained water molecule is not the center of mass, but a different one, e.g. the charge center of the atomistic one.
+
+Run a short MD simulation of a box of SPC/E water using GROMACS. (This is later on needed for the different coarse-graining (CG) methods.) The input files are in the same folder (`csg-tutorials/spce/atomistic/`). Due to limited time, decrease the number of steps (`nsteps`) in `grompp.mdp` to a reasonable value (`5000-10000`). The output frequency of the trajectory, .log and .edr files should then be adjusted, as well to e.g. 50-100 (`nstxout`, `nstfout`, `nstlog`, nstenergy):
+
+```bash
+gmx grompp # combines conf.gro, topol.top, and grompp.mdp, and forcefield.itp (which is in the gromos43a1.ff folder of GROMACS) into topol.tpr
+gmx mdrun -v # runs MD integrator. The trajectory is saved to traj.trr
+```
+
+Map this short atomistic trajectory to a CG trajectory using `csg_map`. (adjust the corresponding line in the `run.sh` script to map `traj.trr` onto `traj_cg.trr` instead of `conf(out).gro` to `conf_cg.gro`.)
+
+
+# Iterative Boltzmann inversion (IBI) for SPC/E water
+
+Here, a one-site coarse-grained (CG) model of a rigid 3-site water molecule (SPC/E model) is constructed (see the previous section of the tutorial) using the iterative Boltzmann inversion (IBI) method. The center of the CG bead is chosen to be the center of mass (COM) of a molecule. The CG radial distribution function (RDF) of the atomistic reference trajectory is then calculated using the CG bead coordinates. In the last step a coarse-grained potential is obtained by matching the RDFs of the atomisitc and CG systems using the IBI method. For a more detailed description, look at the following [publication](http://pubs.acs.org/doi/abs/10.1021/ct900369w).
```

Comment:
"The CG radial distribution function (RDF) of the atomistic reference trajectory is then calculated using the CG bead coordinates." - > "The target radial distribution function (RDF) is calculated from the CG bead coordinates obtained by mapping the reference atomistic trajectory."


" In the last step a coarse-grained ..." -> "In the last step, a coarse-grained ..."